### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by xfq**
Trivial follow-up for #5265.

### CODE REVIEW

File: docs/_appendix/github-intro.md [hunk 1]
```diff
@@ -56,9 +56,7 @@ Now that you have Git set up, you will need to fork the test repository. This
 will enable you to [submit][submit] your tests using a pull request (more on this
 [below][submit]).
 
-1.  In the browser, go the the GitHub page for the test repository:
-
-    Main W3C test repository: [github.com/w3c/web-platform-tests][main-repo]
+1.  In the browser, go the the GitHub page for the test repository: [github.com/w3c/web-platform-tests][main-repo]
```
Comment (annevk) [inline docs/_appendix/github-intro.md]: I think we could make this read even better:

`In the browser, go to [web-platform-tests on GitHub](link)`

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/216667967)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 4a475c4f6d4ed93c0b1bcae7368408dcaea6cfd6
Using browser at version BuildID 20170329100319; SourceStamp 272ce6c2572164f5f6a9fba2a980ba9ccf50770c
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/216667969)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 4a475c4f6d4ed93c0b1bcae7368408dcaea6cfd6
Using browser at version 59.0.3053.3 dev
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5268/)

File: docs/_appendix/github-intro.md (hunk 1 above)
Comment (xfq) [inline docs/_appendix/github-intro.md]: Fixed. Thanks!

**Review by annevk — APPROVED** [review]

**Pull request merged**